src/app/stores/helpers/ParseBackend.coffee

```
@@ -143,6 +143,39 @@ class ParseBackend
           result.set 'favorites', favorites
           result.save()
 
+  deleteFavorite: (userId, cardId) ->
+    log.info "ParseBackend.deleteFavorite(): ", userId, cardId
+    user = new Parse.Object 'User'
+    user.set 'id',  userId
+    card = new Parse.Object 'Card'
+    card.set 'id',  cardId
+    favoriteQuery = new Parse.Query 'Favorite'
+    favoriteQuery.equalTo 'user', user
+    favoriteQuery.equalTo 'card', card
+    favoriteQuery.first
+      success: (result) ->
+        result.destroy
+          success: (object) ->
+
+          error: (object, error) ->
+            log.info error.code, error.message
+      error: (error) ->
+        log.info error.code, error.message
+        alert "Error in delete query"
```

We should use log.error instead of alert here. Fail silently, if not gracefully, for users, but gives devs a way to investigate. Also log.error instead of log.info in 161.